From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Fri, 18 Dec 2020 16:56:38 +0100
Subject: ANDROID: GKI: remove CONFIG_CRC8=y

This is no longer needed, thanks to 4c9398822106 ("PCI: qcom: Add
support for configuring BDF to SID mapping for SM8250") which adds a
"select CRC8" to the CONFIG_PCIE_QCOM option.

Fixes: 4c9398822106 ("PCI: qcom: Add support for configuring BDF to SID mapping for SM8250")
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I739c9e7f152c253e1037eb57aa5fdf6b3d902d5f
---
 arch/arm64/configs/gki_defconfig     | 1 -
 arch/arm64/configs/rockpi4_defconfig | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -575,7 +575,6 @@ CONFIG_CRYPTO_LZ4=y
 CONFIG_CRYPTO_ZSTD=y
 CONFIG_CRYPTO_ANSI_CPRNG=y
 CONFIG_CRC_CCITT=y
-CONFIG_CRC8=y
 CONFIG_XZ_DEC=y
 CONFIG_DMA_CMA=y
 CONFIG_PRINTK_TIME=y
diff --git a/arch/arm64/configs/rockpi4_defconfig b/arch/arm64/configs/rockpi4_defconfig
--- a/arch/arm64/configs/rockpi4_defconfig
+++ b/arch/arm64/configs/rockpi4_defconfig
@@ -519,7 +519,6 @@ CONFIG_CRYPTO_ANSI_CPRNG=y
 CONFIG_CRYPTO_DEV_ROCKCHIP=y
 CONFIG_CRYPTO_DEV_VIRTIO=y
 CONFIG_CRC_CCITT=y
-CONFIG_CRC8=y
 CONFIG_XZ_DEC=y
 CONFIG_DMA_CMA=y
 CONFIG_PRINTK_TIME=y
